src/homepage/howTo/custom-listeners.html

```diff
@@ -0,0 +1,237 @@
+$ mkdir extensions-jars
+$ cd james-project
+$ mvn clean install -DskipTests -T 1C
+$ cd ../
+$ cp james-project/examples/custom-listeners/target/custom-listeners-****.jar extension-jars/custom-listeners.jar
+                </code></pre>
+                <p>
+                    Check out the docker-compose.yml to get the host name of james service, currently it's <b>james.local</b>.
+                    So you, have to modify James domain list to use this <b>james.local</b> as James default domain:
+                </p>
+                <pre><code>
+&lt;domainlist&gt;
+    &lt;autodetect&gt;true&lt;/autodetect&gt;
+    &lt;autodetectIP&gt;true&lt;/autodetectIP&gt;
+    &lt;defaultDomain&gt;james.local&lt;/defaultDomain&gt;
+&lt;/domainlist&gt;
+                </code></pre>
+                <p>
+                    Now, you can able to start james by docker compose, and wait all docker containers up.
+                </p>
+                <pre><code>
+$ docker-compose up -d
+                </code></pre>
+
+                <header class="major">
+                    <h3><b>Verifying the result of the setup</b></h3>
+                </header>
+                <p>
+                    After a long way to setup a proper James server on your local machine, we are able to use James with custom listeners loaded.
+                    To verify our work, here is the simple scenario:
+                </p>
+                <ul>
+                    <li>Use James CLI create user1@james.local/password1</li>
+                    <li>Use James CLI create user2@james.local/password2</li>
+                    <li>Use a mail client to connect to user1, send a big email to use2</li>
+                    <li>Use IMAP command to see the flags of the email user2 had received</li>
+                </ul>
+
+                <p>Use James CLI to provision users</p>
+                <pre><code>
+$ docker exec -it james /bin/bash
+$ java -jar james-cli.jar AddUser user1@james.local password1
+$ java -jar james-cli.jar AddUser user2@james.local password2
+$ exit
+                </code></pre>
+
+                <p>
+                    Use thunderbird or any mail client on your local machine to connect with user1 and user2.
+                    The configuration should point to smpt server at <code>localhost:25</code>, and imap server at <code>localhost:143</code>.
+                    Use these credentials: user1@james.local/password1 and user2@james.local/password2
+                </p>
+                <p>
+                    After that, use user1@james.local account to sent a big email to user2@james.local, the size of the emails should be greater than 1 MB.
+                    Take any picture that's bigger than 1 MB then attach it to the mail is one of the simplest ways.
+                </p>
+                <p>
+                    If the mail client had sent a mail to user2@james.local, then if you can see that email appeared in user2 INBOX,
```

Comment:
If the mail client had sent a mail to user2@james.local, then if you can see that email appeared in user2 INBOX,
we can go to see its flags by using IMAP command at your local host 
=>
  If the mail client had sent a mail to user2@james.local, then you can see that this email appeared in user2 INBOX.
  We can see its flags by using the following IMAP command on your local host.